notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -10483,6 +10483,64 @@ drug abuse [Linda was abusing alcohol and the happiness-producing drug Soma.]
 coping with being famous [John Cooper despised being a media sensation.]
 
 
+movie: Deep Impact (1998)
+=========================
+
+:: Title
+Deep Impact
+
+:: Description
+The film depicts the attempts to prepare for and destroy a 7-mile wide comet
+set to collide with Earth and cause a mass extinction.
+
+:: Date
+1998-05-08
+
+:: References
+https://en.wikipedia.org/wiki/Deep_Impact_(film)
+
+:: Ratings
+2 <paul>
+
+:: Genre
+sci-fi
+
+:: Choice Themes
+approaching astronomical object event [The main novelty of the film is that a roughly Mr. Everest sized comet is discovered to be on a collision course for Earth.]
+
+:: Major Themes
+journalism [One storyline follows the MSNBC reporter Jenny Lerner as she covers the biggest story in history: humanity's efforts to stop a giant comet from striking Earth.]
+speculative space travel [The United States and Russia sent a joint team of astronauts on a dangerous mission to fly out to intercept the comet and alter its course with nuclear bombs.]
+nuclear powered spaceship [The United States and Russia worked together to construct the Messiah, an experimental nuclear powered vessel loosely patterned on the space shuttle, in orbit and a team of astronauts subsequently fly it out to intercept the comet.]
+the calculus of human life [Once it became apparent that the comet was going to strike the earth, the United States government selected 800,000 Americans by lottery to join 200,000 pre-selected individuals in underground shelters. Everyone else was written off for dead.]
+facing death [All but a select few faced the prospect that they were going to die horribly once the comets hit Earth.]
+what it is like during a natural disaster [The viewer is shown how the build up to an extinction level event cometary impact might unfold.]
+father and daughter [One storyline follows Jenny and her estranged father as he tries to make things right between them.]
+reconciliation [A Mt. Everest sized comet careening toward Earth proved to be just the kick in the pants that Jenny her estranged father, Jason, needed to heal their toxic relationship. In the end, they pointedly embraced on the seashore and were subsequently obliterated by an oncoming 1,200 foot tidal wave.]
+boyfriend and girlfriend [One storyline follows the teen lovers Leo and Sarah as they try to find a way to both survive the comet strike.]
+teen romance [One storyline follows the teen lovers Leo and Sarah as they try to find a way to both survive the comet strike.]
+
+:: Minor Themes
+astronomy [The teenage amateur astronomer Leo Beiderman spotted an unusual celestial object in the sky during a star viewing party.]
+alcohol abuse [The viewer learns that Mrs. Rittenhouse had taken to the bottle in the wake of her husband having a series of affairs, and had subsequently "stepped it up last summer" after he son died of leukemia.]
```

Comment:
Typo...

Reply:
Fixed and pushed to dev.

Reply:
And cherry picked to this movie-deepimpact-1998 branch.